server/src/main/java/org/elasticsearch/common/logging/Loggers.java

```diff
@@ -126,7 +112,6 @@ private static String formatPrefix(String... prefixes) {
                 }
             }
             if (sb.length() > 0) {
-                sb.append(" ");
```

Comment:
This was required because our logger format didn't always have a prefix. The space is now included in the new logger format.